### DISCUSSION TITLE
Please implement 2FA

### DISCUSSION THREAD

**Discussion by halvar51** [paperless-ngx/paperless-ngx#679] [Feature Requests]
Since sensitive documents may be stored here, the use of 2FA would be very welcome.

**Comment by shamoon** [comment]
See #424 #435 also #211

**Comment by stepanov1975** [comment]
Just use Authelia - https://www.authelia.com/

**Comment by pandruszkow** [comment]
You should use an external application and/or a reverse proxy for this.
